packages/plan-form-core/package.json

```diff
@@ -36,12 +36,10 @@
   },
   "author": "OpenSRP Engineering",
   "license": "Apache-2.0",
-  "private": false,
   "dependencies": {
     "@onaio/utils": "^0.0.1",
     "date-fns": "2.11.1",
     "lodash": "^4.17.20",
-    "moment": "^2.29.1",
```

Comment:
this is being used in the package

Reply:
[42](https://github.com/opensrp/web/pull/961/commits/ecf01422e11a4f8d8a1c0aad04ea69d0b2ffb652)